### PR TITLE
[chore] Release 0.116.0

### PR DESCRIPTION
**Pull request by jaronoff97**
Release for 0.116.0 using image version 0.116.1

### CODE REVIEW

File: versions.txt [hunk 1]
```diff
@@ -34,8 +34,8 @@ autoinstrumentation-go=v0.19.0-alpha
 
 # Represents the current release of Apache HTTPD instrumentation.
 # Should match autoinstrumentation/apache-httpd/version.txt
-autoinstrumentation-apache-httpd=1.0.4
```
Comment (swiatekm) [inline versions.txt:37]: Why are you bumping this? There's a PR for it [here](https://github.com/open-telemetry/opentelemetry-operator/pull/3488) and it looks like it's not clear what is actually supported.

Comment (jaronoff97) [reply versions.txt:37]: argh ill undo...

File: autoinstrumentation/python/requirements.txt [hunk 2]
```diff
@@ -1,61 +1,61 @@
-opentelemetry-distro==0.48b0
+opentelemetry-distro==0.50b0
 # We don't use the distro[otlp] option which automatically includes exporters since gRPC is not appropriate for
 # injected auto-instrumentation, where it has a strict dependency on the OS / Python version the artifact is built for.
 opentelemetry-exporter-otlp-proto-http==1.27.0
-opentelemetry-exporter-prometheus==0.48b0
```
Comment (swiatekm) [inline autoinstrumentation/python/requirements.txt:5]: I'm not sure I like doing this in a release PR.

Comment (jaronoff97) [reply autoinstrumentation/python/requirements.txt:5]: yeah it's decently out to date... i can leave it out though if you prefer.

Comment (pavolloffay) [reply autoinstrumentation/python/requirements.txt:5]: I would prefer to be done this in a separate PR as well

File: versions.txt [hunk 3]
```diff
@@ -2,16 +2,16 @@
...
 # Represents the current release of the Target Allocator.
-targetallocator=0.115.0
+targetallocator=0.116.0
 
 # Represents the current release of the Operator OpAMP Bridge.
-operator-opamp-bridge=0.115.0
+operator-opamp-bridge=0.116.0
 
 # Represents the current release of Java instrumentation.
 # Should match autoinstrumentation/java/version.txt
```
Comment (pavolloffay) [inline versions.txt:17]: Java image should be bumped

File: autoinstrumentation/python/requirements.txt (hunk 2 above)
Comment (pavolloffay) [reply autoinstrumentation/python/requirements.txt:5]: It should be done in a separate PR that will publish the python image and then use that image in this PR.

File: versions.txt (hunk 3 above)
Comment (TylerHelmuth) [reply versions.txt:17]: Yes java is at 1.33.6 now

**Review by yuriolisa — APPROVED** [review]

**Comment by pavolloffay** [comment]
The CI is failing :/

**Comment by swiatekm** [comment]
> The CI is failing :/

Yeah, most likely caused by changes to tests in https://github.com/open-telemetry/opentelemetry-operator/pull/3494. I tried to fix them in https://github.com/open-telemetry/opentelemetry-operator/pull/3560, but it clearly wasn't enough. Maybe we should just skip for now?

**Comment by swiatekm** [comment]
I think the most problematic test flakiness is fixed in https://github.com/open-telemetry/opentelemetry-operator/pull/3564.

**Pull request merged**